.template/addons/docker/Dockerfile.tt

```diff
@@ -23,7 +23,7 @@ ENV BUILD_ENV=$BUILD_ENV \
     <%- if WEB_VARIANT -%>
     ASSET_HOST=$ASSET_HOST \
     NODE_ENV=$NODE_ENV \
-    NODE_VERSION="12" \
+    NODE_VERSION=<%= NODE_SOURCE_VERSION %> \
```

Comment:
How about change it to 
```suggestion
    NODE_SOURCE_VERSION=<%= NODE_SOURCE_VERSION %> \
```
?

Reply:
it's kind of confused between NODE_VERSION and NODE_SOURCE_VERSION

Reply:
Hi @namtx 

- NODE_VERSION is the nvm version (used for `.nvmrc`) 
- NODE_SOURCE_VERSION is the NodeSource Node.js Binary Distributions used to install Node in the Docker image.

Let me try to use the same value (12) for `nvm` if it works or not.

Reply:
Setting the `.nvmrc` with `12` doesn't work as expected @namtx

![image](https://user-images.githubusercontent.com/11751745/91249887-eced6600-e781-11ea-8dd5-3257d5ec2fb9.png)

Reply:
I mean 
```
NODE_SOURCE_VERSION=<%= NODE_SOURCE_VERSION %>
```
it makes me confused because, in our template, we're using `NODE_VERSION` for assigning the node version but in Dockerfile, we're using it as `NODE_SOURCE_VERSION`

Reply:
It's because

On development env, we are using nvm to control the Node version because we might have many different Node versions depend on each project, 

But in the Docker, we just need to use only 1 Node version so there is no point to setup `nvm` there, just install the Node directly.

Do you have other ideas to make it clear? 🤔

Reply:
@anduonghien like I commented, how do we rename in NODE_VERSION to NODE_SOURCE_VERSION in Dockerfile?

Reply:
https://github.com/nimblehq/rails-templates/pull/207/files#r476995350
the node version in the nvm file and Dockerfile is not related.

Reply:
Oh got it @namtx I misunderstanding your comment before, changed it in https://github.com/nimblehq/rails-templates/pull/207/commits/5a57ba42d51002c0c8e84a45ee2b90f88c271d88